addons/extensions/hive-mirroring/src/main/resources/runtime/hive-mirroring-secure-workflow.xml

```diff
@@ -142,10 +142,8 @@
             <arg>${clusterForJobRunWriteEP}</arg>
             <arg>-clusterForJobNNKerberosPrincipal</arg>
             <arg>${clusterForJobNNKerberosPrincipal}</arg>
-            <arg>-tdeEncryptionEnabled</arg>
```

Comment:
TDE encryption attributes removed from secure mirroring workflow. Is this expected.